jest-puppeteer.config.js

```diff
@@ -10,6 +10,12 @@ const { BASE_URL, HEADLESS, PORT = '3000' } = process.env
 module.exports = {
   browserContext: 'incognito',
 
+  /**
+   * Workaround for jest-environment-puppeteer 'uncaughtException'
+   * see error handling in '@nhsuk/frontend-helpers/jest/environment/jest.puppeteer.mjs'
+   */
+  exitOnPageError: false,
```

Comment:
Copied from GOV.UK Frontend

Effectively tells Jest Puppeteer we're handling our own exceptions